packages/cli-dxns/src/dxos.proto

```diff
@@ -0,0 +1,79 @@
+syntax = "proto3";
+
+package dxos;
+
+message DXNS_cid {
+  bytes data = 1;
+}
+
+message IPFS_cid {
+  bytes data = 1;
+}
+
+message Any {
+  DXNS_cid schema_cid = 1;
+  string mesage_fqn = 2;
+  bytes payload = 3;
+}
+
+message RecordAttributes {
+  string name = 1;
+  string description = 2;
+  string version = 3;
+  string author = 4;
+}
+
+message KUBE {
+  RecordAttributes attributes = 1;
+  string url = 2;
+
+  Any extension = 10;
+}
+
+message Service {
+  RecordAttributes attributes = 1;
+  string type = 2;
+
+  Any extension = 10;
+}
+
+message App {
+  RecordAttributes attributes = 1;
+  string build = 2;
```

Comment:
How is this different from `version`? If we use semver for versions it has support for build numbers in the standard: `1.2.3+abc123buildnunber`